### PR TITLE
BLACK Token submission

### PR DESCRIPTION
**Pull request by ancianoencrypto**
# Pull Request Template

## Description

Registration request for BLACK Token

## Type of change

- [x] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

File: mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            }
+        ]
+    },
+    "policy": "82018201828200581cf950845fdf374bba64605f96a9d5940890cc2bb92c4b5b55139cc00982051a07301eac",
+    "description": {
+        "sequenceNumber": 0,
+        "value": "The blackest shitcoin on Cardano - In BlackSnek We Trust",
+        "signatures": [
+            {
+                "signature": "7250b04b4eac64222bc55483123ecfd3cd248ae583aeba540f2877fb9f7b306670d8e48676fafd99fdfb85bfa871565d9f71db108004c9022e31e1f73b8d5d02",
```
Comment (VladislavKudrin) [inline mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json:49]: Hi @ancianoencrypto,

the signature for the description is failing. Please, resign it and submit again

Thanks!

Comment (ancianoencrypto) [reply mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json:49]: What is the error?

Comment (VladislavKudrin) [reply mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json:49]: @ancianoencrypto,

`Failed to validate wallet metadata entry, error was: 'attestation verification failed for: description.`

Which indicates that the signature for the description field is not valid. You can resign the file again and update the PR. 
Also a tip - you can check the file before submitting to the registry locally, you should run:
`token-metadata-creator validate {subject}.json`

Comment (ancianoencrypto) [reply mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json:49]: I do not know how to modify it

Comment (VladislavKudrin) [reply mappings/36f9e394b63bac4fd8aae236adb6fb5a45aac2d774ab220b96226eee424c41434b.json:49]: @ancianoencrypto

you don't need to modify it. Simply create another draft and sign it.

[How](https://github.com/cardano-foundation/cardano-token-registry/wiki/How-to-prepare-an-entry-for-the-registry-%28NA-policy-script%29)

**Comment by VladislavKudrin** [comment]
Closed, because due to inactivity

**Pull request closed without merging**